src/main/resources/lang/lang_de_DE.properties

```diff
@@ -1074,3 +1074,4 @@ CommandFactionJoinFactionFull=Die Fraktion, der du beitreten m
 CommandFactionAddMemberTargetFactionFull=Die Fraktion, zu der du diesen Spieler hinzuf�gen m�chtest, ist voll.
 PlayerInteractEntityFailedToSavePlayer=Der Spieler konnte nicht gespeichert werden.
 PlayerInteractEntityCannotTradeWithVillager=Du kannst nicht mit diesem Dorfbewohner handeln.
+CommandFactionClaimFillTooManyChunks=Sie k�nnen nicht mehr als {0} Chunks gleichzeitig beanspruchen.
```

Comment:
Should use impolite form for consistency (Du kannst nicht mehr als {0} Chunks gleichzeitig beanspruchen.)

Reply:
We're now using impolite form here :) Thanks for this!